config/filter.d/sshd.conf

```diff
@@ -18,7 +18,7 @@ before = common.conf
 
 _daemon = sshd
 
-failregex = ^%(__prefix_line)s(?:error: PAM: )?[aA]uthentication (?:failure|error) for .* from <HOST>( via \S+)?\s*$
+failregex = ^%(__prefix_line)s(?:error: PAM: )?[aA]uthentication (?:failed|failure|error) for .* from <HOST>( via \S+)?\s*$
```

Comment:
I tend to merge it, but would prefer another order here (because all other OS have `failURE`)...
So please change to 
```
...(?:failure|error|failed)...
```

Thx